frontend/app/routes/teams/index.js

```diff
@@ -34,6 +34,16 @@ export default class TeamsIndexRoute extends Route {
     }
   }
 
+  beforeModel(transition) {
+    let definedParamValues = Object.values(transition.to.queryParams).filter(
+      value => !!value
```

Comment:
use isPresent?

Reply:
Thought the same, checked their docs, doesn't work.